data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
+			`	She is disrupted when she hears a loud roar, and jumps to the ground to narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`
+			`	Having dodged the pounce, she runs past the bird, looking for cover, but the forest isn't dense enough to block the beast's advance. It attacks again, and wounds the top of one of her frontal legs. She kicks it with another leg, and readies an arrow, but it only scratches its face. The animal gets on its hind legs to swipe down at her, but is stunned, for the bird has attacked its eyes, blinding it. The huntress is swift to take aim, and an arrow pierces the creature's torso. Once it's fallen the bird flies away, and the curtains close.`
+				goto end
+			label seed
+			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and you do so.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
+			`	The central figure is a young Saryd boy, galloping in a dense forest. Sound effects of thunder begin and grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`
+			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him. He goes to it and lays down at the base of that largest tree, and soon he falls asleep as the curtain closes.`
```

Comment:
```suggestion
			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him. He goes to it and lays down at its base, and soon he falls asleep as the curtain closes.`
```
An alternative  would be "at the base of that largest of trees". The suggestion improves readability, the comment emphasizes that the tree is special.